server/springboot/src/main/java/com/styra/tickethub_springboot/dao/model/SecurityConfig.java

```diff
@@ -132,7 +136,14 @@ AuthorizationManager<RequestAuthorizationContext> customAuthManager() {
 
             //OPAClient opa = new OPAClient(opaURL);
 
-            HTTPClient client = new LatencyMeasuringHTTPClient();
+            Logger logger = Logger.getLogger("mylogger");
```

Comment:
none of this should be necessary

Reply:
This was a PoC, the actual item is over here https://github.com/StyraInc/opa-java/pull/55

Reply:
Resolving this as discussion has moved to opa-java.